platform/security/sts/security-sts-certificateclaimshandler/src/main/java/org/codice/ddf/security/claims/certificate/CertificateClaimsHandler.java

```diff
@@ -0,0 +1,122 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.claims.certificate;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.Principal;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.cxf.common.util.StringUtils;
+import org.apache.cxf.rt.security.claims.Claim;
+import org.apache.cxf.rt.security.claims.ClaimCollection;
+import org.apache.cxf.sts.claims.ClaimsHandler;
+import org.apache.cxf.sts.claims.ClaimsParameters;
+import org.apache.cxf.sts.claims.ProcessedClaim;
+import org.apache.cxf.sts.claims.ProcessedClaimCollection;
+import org.apache.cxf.sts.token.realm.RealmSupport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CertificateClaimsHandler implements ClaimsHandler, RealmSupport {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CertificateClaimsHandler.class);
+
+    static final String EMAILADDRESS =
+            "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress";
+
+    static final String COUNTRY = "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/country";
+
+    private String emailClaim = EMAILADDRESS;
+
+    private String countryClaim = COUNTRY;
+
+    private List<String> supportedRealms;
+
+    private String realm;
+
+    @Override
+    public List<URI> getSupportedClaimTypes() {
+        try {
+            return Arrays.asList(new URI(emailClaim), new URI(countryClaim));
+        } catch (URISyntaxException e) {
+            LOGGER.info("Unable to create email address claim URI for certificate claims.", e);
+        }
+        return Collections.emptyList();
+    }
+
+    @Override
+    public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
+            ClaimsParameters parameters) {
+        ProcessedClaimCollection claimsColl = new ProcessedClaimCollection();
+        Principal principal = parameters.getPrincipal();
+        Map<String, Object> additionalProperties = parameters.getAdditionalProperties();
+        if (additionalProperties != null && (additionalProperties.containsKey(EMAILADDRESS)
+                || additionalProperties.containsKey(COUNTRY))) {
+            for (Claim claim : claims) {
+                URI claimType = claim.getClaimType();
+                if (emailClaim.equals(claimType.toString())) {
+                    buildClaim(claimsColl,
```

Comment:
Did these two calls to `buildClaim` really format differently? Is it the length of the const making the difference?